src/lib/Timer.ts

```diff
@@ -0,0 +1,85 @@
+import { chalk as console } from "./coloredConsole"
+import marky = require("marky")
+
+/**
+ * Timer to mesure performance in wdi5 or TS apps
+ */
+
+let active = true
+
+/**
+ * do not use the Timer Class
```

Comment:
erm, what?
it's difficult to grasp the concept behind this timer file just by the src → some "why", "how" and "@example" comments/JSDoc would help tremendously here

Reply:
jup, for discussion. Not sure if we actually have any benefits of marky vs. `console.time`.

Reply:
intention would be to keep (our) custom code as sparse as possible → the more we can achieve by (re-)using `marky`, the better